msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -67,10 +87,10 @@ The [get members](/microsoftteams/platform/bots/how-to/get-teams-context#fetch-t
 | `id` | Unique generated value for the anonymous user. |
 | `name` | Name provided by the anonymous user when joining the meeting. |
 | `tenantId` | Tenant ID of the meeting organizer. |
-| `userRole` | `anonymous`, represents anonymous user. |
+| `userRole` | `anonymous` represents anonymous user. |
 
 > [!NOTE]
-> The ID received from the bot APIs and the Teams client library API are not the same.
+> The ID received from the bot APIs and the Teams client library API aren't the same.
```

Comment:
There's no user ID anymore in tab context, no?

Reply:
That's correct. Thanks for the callout, please remove.